src/main/duke/command/AddCommand.java

```diff
@@ -0,0 +1,79 @@
+package duke.command;
+
+import duke.DukeException;
+import duke.Storage;
+import duke.TaskList;
+import duke.Ui;
+import duke.task.DeadlineTask;
+import duke.task.EventTask;
+import duke.task.Task;
+import duke.task.TodoTask;
+
+public class AddCommand extends Command {
+
+    private Task taskToAdd;
+    
+    public AddCommand(String fullCommand) throws DukeException{
+        parseTask(fullCommand);
+    }
+
+    /**
+     * Differentate which tasks user is adding.
```

Comment:
Short and sweet description, however I noticed a similar issue elsewhere, where the description and arguments were missing a line in between.